apps/web-giddh/src/app/vouchers/create/create.component.ts

```diff
@@ -2064,8 +2064,19 @@ export class VoucherCreateComponent implements OnInit, OnDestroy, AfterViewInit
         this.invoiceForm.controls[entityType].get(addressType).get("address").patchValue(typeof address?.address === "string" ? [address?.address] : address?.address);
         this.invoiceForm.controls[entityType].get(addressType).get("pincode").patchValue(address?.pincode);
         this.invoiceForm.controls[entityType].get(addressType).get("taxNumber").patchValue(address?.gstNumber || address?.taxNumber);
-        this.invoiceForm.controls[entityType].get(addressType).get("state").get("name").patchValue(address?.state?.name ? address?.state?.name : address?.stateName ? address?.stateName : "");
-        this.invoiceForm.controls[entityType].get(addressType).get("state").get("code").patchValue(address?.state?.code ? address?.state?.code : address?.stateCode ? address?.stateCode : "");
+        const state = { name: "", code: "" };
+        if (address?.state) {
+            state.name = address.state?.name;
+            state.code = address.state?.code;
+        } else if (address?.stateName && address?.stateCode) {
```

Comment:
Please perform unit testing for the stateName and stateCode."

Reply:
Done